zerver/data_import/slack.py

```diff
@@ -452,14 +461,18 @@ def process_customprofilefields(
                 break
 
 
+def is_slackbot(user: ZerverFieldsT) -> bool:
+    return get_user_full_name(user).lower() == "slackbot"
+
+
 def get_user_email(user: ZerverFieldsT, domain_name: str) -> str:
     if "email" in user["profile"]:
         return user["profile"]["email"]
     if user["is_mirror_dummy"]:
         return Address(username=user["name"], domain=f"{user['team_domain']}.slack.com").addr_spec
     if "bot_id" in user["profile"]:
         return SlackBotEmail.get_email(user["profile"], domain_name)
-    if get_user_full_name(user).lower() == "slackbot":
+    if is_slackbot(user):
```

Comment:
we already had this heuristic for identifying the Slackbot account in the data. we can switch to detecting by the `USLACKBOT` id as suggested by @amanagr if that feels better, but i went with keeping the method we've been relying on for now.